scripts/setup-manylinux.sh

```diff
@@ -262,7 +262,21 @@ function install_arrow {
 
 function install_cuda {
   # See https://developer.nvidia.com/cuda-downloads
-  dnf config-manager --add-repo https://developer.download.nvidia.com/compute/cuda/repos/rhel9/x86_64/cuda-rhel9.repo
+  #dnf config-manager --add-repo https://developer.download.nvidia.com/compute/cuda/repos/rhel9/x86_64/cuda-rhel9.repo
+  local arch=$(uname -m)
+  local repo_url
+
+  if [[ "$arch" == "x86_64" ]]; then
+    repo_url="https://developer.download.nvidia.com/compute/cuda/repos/rhel9/x86_64/cuda-rhel9.repo"
+  elif [[ "$arch" == "aarch64" ]]; then
+    # Using SBSA (Server Base System Architecture) repository for ARM64 servers
```

Comment:
This is a near copy of the changes in the centos9 script but this copy has a comment. It is probably worth matching (with comment both places).